### PR TITLE
Build enhancement

### PR DESCRIPTION
**Pull request by imbcmdth**
Output (`dist/videojs.hls.js`) includes:
* `videojs-contrib-media-sources` with `mux.js` as an inline blob webworker
* `videojs-contrib-hls`

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -24,17 +24,11 @@ module.exports = function(grunt) {
       },
       dist: {
         nonull: true,
-        src: ['src/videojs-hls.js',
+        src: ['node_modules/videojs-contrib-media-sources/dist/videojs-media-sources.js',
```
Comment (gkatsev) [inline Gruntfile.js]: I still hate this so much.

Comment (gkatsev) [inline Gruntfile.js]: (The fact that we need to list out all our files, that is).

Comment (forbesjo) [inline Gruntfile.js]: Would browserify make this simplier?

Comment (gkatsev) [inline Gruntfile.js]: Yes.

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,4 @@
 .DS_Store
-dist/*
```
Comment (forbesjo) [inline .gitignore]: Why don't shouldn't the dist just live on npm?

Comment (gkatsev) [inline .gitignore]: Yeah, re-add it here and get rid of it in `.npmignore`.

**Pull request closed without merging**